aten/src/ATen/cuda/detail/PhiloxCudaStateRaw.cuh

```diff
@@ -0,0 +1,38 @@
+// Stores RNG state values. Passed as a kernel argument. See Note [CUDA Graph-safe RNG states].
```

Comment:
I'd add a comment here explaining why this "header" doesn't have an include guard (or #pragma once)